test/test_integration_credentials.py

```diff
@@ -10,12 +10,15 @@
 import shutil
```

Comment:
I would name it test_integration_credentials_keyring because the other file is _netrc